data_api/staging/admin.py

```diff
@@ -9,10 +21,23 @@ class SyncCheckpointAdmin(admin.ModelAdmin):
     list_filter = ['is_running', 'organization', 'collection_name', 'last_started', 'last_saved']
 
 
-class OrganizationAdmin(admin.ModelAdmin):
+class OrganizationAdmin(ExtraUrlMixin, admin.ModelAdmin):
     list_display = ['name', 'country', 'is_active', 'server', 'api_token']
     list_filter = ['is_active', 'server']
 
+    @action()
+    def _sync_organization(self, request, pk):
+        organization = get_object_or_404(Organization, pk=pk)
+        client = TembaClient(organization.server, organization.api_token)
+        import_org_with_client(client, organization.server, organization.api_token)
+        return HttpResponseRedirect(reverse('admin:staging_organization_change', args=[organization.pk]))
+
+    @action()
+    def _sync_latest_data(self, request, pk):
+        organization = get_object_or_404(Organization, pk=pk)
+        sync_latest_data(entities=None, orgs=[organization.api_token])
```

Comment:
I think we should call this asynchronously since it could take hours or longer to complete.

Reply:
what do you mean with screenshot?
like a screenshot file in the repo?